cypress/integration/brewery/CreateScenario-tests.spec.js

```diff
@@ -134,7 +192,7 @@ describe('Create scenario', () => {
     cy.intercept('PATCH', URL_REGEX.SCENARIO_PAGE_WITH_ID).as('requestEditScenario');
     cy.intercept('POST', URL_REGEX.SCENARIO_PAGE_RUN_WITH_ID).as('requestRunScenario');
 
-    ScenarioParameters.updateAndLaunch();
+    ScenarioParameters.updateAndLaunch(true);
```

Comment:
Can we remove all modifications `ScenarioParameters.updateAndLaunch(true);` to use a default value instead ? (see previous comment)
Note that when only one scenario is launched in a test block, the local storage might be removed when switching to the next test, so trying to remember the "don't ask again" value might be pointless here.